examples/README.md

```diff
@@ -96,3 +95,15 @@ We use [SemVer](http://semver.org/) for versioning. For the versions available,
 Most Grad IT projects will belong to ABOR, so use the following (be sure to update the year as needed):
 
 Copyright (c) 2019 Arizona Board of Regents on behalf of the University of Arizona - All Rights Reserved.
+
+## Resources (optional)
+
+Resources can include things like websites that help to figure out how to do something.
```

Comment:
Resources can include things like websites that help to figure out how to do something

change to:

Resources can include things like websites that help you figure out how to do something